meson.build

```diff
@@ -186,3 +243,11 @@ datapaths = [
 foreach dp: datapaths
   install_subdir(dp, install_dir: 'share/games/performous')
 endforeach
+
+#TBD: desktop file, pixmap files, man etc. see cmake of data and docs dir
```

Comment:
Please make a separate issue for this or make it usable

Reply:
We'll follow on the missing bits in a separate issue indeed

Reply:
Issue is in: https://github.com/performous/performous/issues/604